examples/api/lib/material/expansion_tile/expansion_tile.0.dart

```diff
@@ -6,33 +6,30 @@
 
 import 'package:flutter/material.dart';
 
-void main() => runApp(const MyApp());
+void main() => runApp(const ExpansionTileApp());
 
-class MyApp extends StatelessWidget {
-  const MyApp({super.key});
-
-  static const String _title = 'Flutter Code Sample';
+class ExpansionTileApp extends StatelessWidget {
+  const ExpansionTileApp({super.key});
 
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
```

Comment:
Please specify 
`theme: ThemeData(useMaterial3: true),`
 
We shouldn't be demonstrating M2 anymore.